pkg/monitor/api/types.go

```diff
@@ -50,6 +50,10 @@ const (
 	// which corresponds to policy_verdict_notify defined in bpf/lib/policy_log.h
 	MessageTypePolicyVerdict
 
+	// MessageTypeRecCapture is a BPF datapath notification carrying a RecorderCapture
+	// which corresponds to capture_msg defined in bpf/lib/pcap.h
+	MessageTypeRecCapture
```

Comment:
Please also add a `MessageTypeNameRecCapture` below (line 66+) and add it to `MessageTypeNames`

This the name (e.g. `record`) will allow you to filter it out in the CLI via e.g. `cilium monitor -t record` in the `m.match` statement you added in `pkg/monitor/format`